pulpcore/cli/common/generic.py

```diff
@@ -232,7 +232,7 @@ def _callback(
 def _href_callback(
     ctx: click.Context, param: click.Parameter, value: Optional[str]
 ) -> Optional[str]:
-    if value is not None:
+    if value:
```

Comment:
I don't think, this is the right location. Instead we should hook into the automatic lazy lookup machinery.

Reply:
What is the purpose of setting "" (i.e., string null) value to a variable if we definitely expect href to be a non-null string? Because that is what we are doing right now. `is not None` is evaluated to `True` for "".

Reply:
I may add a new condition to https://github.com/pulp/pulp-cli/blob/8aaa08740b4f994559ea64e4c682fe612af67cd8/pulpcore/cli/common/context.py#L405 to prevent the evaluation of "self.show(self._entity_lookup["pulp_href"])", but I do not like it.